fdbserver/TLogGroup.actor.cpp

```diff
@@ -263,6 +281,12 @@ void TLogGroupCollection::seedTLogGroupAssignment(
 		assignStorageTeam(teamId, group);
 		storageTeams[teamId] = serverSrcUID;
 	}
+
+	// Map the split keys to storage team IDs
+	this->splitKeyStorageTeamIDMapping = splitKeyToStorageTeamIDMapper;
+	for (const auto& [splitKey, storageTeamID] : this->splitKeyStorageTeamIDMapping) {
+		tr.set(arena, splitKeyToStorageTeamIDKey(splitKey), BinaryWriter::toValue(storageTeamID, Unversioned()));
```

Comment:
Instead of assigning split keys to teams, we should assign key ranges to teams. I.e., for split keys of "a", "b", and "c", they define 4 ranges: [normalKey.begin, "a"), ["a", "b"), ["b", "c"), ["c", normalKey.end). And I don't think we need to add such a mapping. See my other comments.